xfr.go

```diff
@@ -160,20 +160,18 @@ func (t *Transfer) inIxfr(id uint16, c chan *Envelope) {
 // The server is responsible for sending the correct sequence of RRs through the
 // channel ch.
 func (t *Transfer) Out(w ResponseWriter, q *Msg, ch chan *Envelope) error {
-	r := new(Msg)
-	// Compress?
-	r.SetReply(q)
-	r.Authoritative = true
-
+	w.TsigTimersOnly(true)
```

Comment:
This needs to happen after the first set the RRs have been written, because only then they become used. (have to reread the spec and actually test for it with tsig to see if I'm right here). Just keep it where is was.